odoo/addons/base/models/ir_ui_view.py

```diff
@@ -509,6 +510,8 @@ def create(self, vals_list):
                         values['arch_fs'] = '/'.join(path_info[0:2])
                         values['arch_updated'] = False
             values.update(self._compute_defaults(values))
+            if 'type' in values and values['type'] not in valid_types:
+                raise ValidationError(("Invalid view type: '%s'. Allowed types are: %s") % (values['type'], ', '.join(valid_types)))
```

Comment:
The issue was generated when the user did not provide a view type, and the code tried to set the first element tag as the type, but it was wrong at below line, So you shuld apply this fix here https://github.com/odoo/odoo/blob/d0af4a0aad2f5001060e7e9fd54c4a8b6f35835b/odoo/addons/base/models/ir_ui_view.py#L490

```py
if view_type not in valid_types:
     raise ValidationError(_(
        "Invalid view type: '%(view_type)s'.\n"
        "Allowed types are: %(valid_types)s",
        view_type=view_type, valid_types=', '.join(valid_types)
    ))
```